utils/build/docker/nodejs/install_ddtrace.sh

```diff
@@ -4,25 +4,34 @@ set -eu
 
 cd /usr/app
 
-if [ -e /binaries/nodejs-load-from-npm ]; then
-    target=$(</binaries/nodejs-load-from-npm)
-    echo "install from: $target"
-
-elif [ -e /binaries/dd-trace-js ]; then
-    target=$(npm pack /binaries/dd-trace-js)
-    echo "install from local folder /binaries/dd-trace-js"
+if [ -e /binaries/nodejs-load-from-local ]; then
+    echo "using local version that will be mounted at runtime"
 
+    echo "0.0.0" > SYSTEM_TESTS_LIBRARY_VERSION
```

Comment:
As it'll be always the `dev` version, using a high version number is preferable : otherwise, any test activated "starting" a version will be deactivated locally.

And better if it's possible : get the real version from the local folder.

Reply:
> do they need to be moved to app.sh maybe ?

Unfortunately not possible, it must be done at build step.

Reply:
Good point on getting the real version, which should also be possible for the dependencies I think. However, this does mean that the weblog will need to be rebuilt whenever there is a version change. That is probably an acceptable limitation though, and I prefer that trade-off to the trade-off of using a high version number which could fail in an older branch.

Reply:
#2801 has now landed and I've adapted this PR to use the new health check, along with adding the AppSec event rules versions to the endpoint. So the version is now collected entirely at runtime, no weblog rebuild needed even when changing dd-trace-js branch locally.